### PR TITLE
Feat/#13 started app

### PR DESCRIPTION
**Pull request by ciaranschutte**

### CODE REVIEW

File: components/pages/Applications/Dashboard/Applications/index.tsx [hunk 1]
```diff
@@ -7,10 +7,14 @@ import StartApplication from './Start';
 import { css } from '@emotion/core';
 
 type ApplicationsProps = {
-  inProgressApplications: any;
+  inProgressApplication?: any;
 };
 
-const Applications = ({ inProgressApplications = true }: ApplicationsProps) => {
+const application = { appId: '', state: '', submitterId: '', expiresAtUtc: '', updatedAtUtc: '' };
```
Comment (caravinci) [inline components/pages/Applications/Dashboard/Applications/index.tsx:13]: Guessing this is some temporary initial state, while we integrate things?

Comment (ciaranschutte) [reply components/pages/Applications/Dashboard/Applications/index.tsx:13]: you betcha :)

**Review by caravinci — APPROVED** [review]
Looking real good to squash afaik, unless someone else has anything to add?

File: components/pages/Applications/Dashboard/Applications/index.tsx [hunk 2]
```diff
@@ -7,10 +7,14 @@ import StartApplication from './Start';
 import { css } from '@emotion/core';
 
 type ApplicationsProps = {
-  inProgressApplications: any;
+  inProgressApplication?: any;
```
Comment (samrichca) [inline components/pages/Applications/Dashboard/Applications/index.tsx:10]: do we know the shape/type of this?

File: components/pages/Applications/Dashboard/Applications/index.tsx (hunk 1 above)
Comment (samrichca) [inline components/pages/Applications/Dashboard/Applications/index.tsx:13]: this is identical to initState above - is this temporary? could it be a shared/exported constant?

File: components/pages/Applications/Dashboard/Applications/index.tsx (hunk 2 above)
Comment (ciaranschutte) [reply components/pages/Applications/Dashboard/Applications/index.tsx:10]: @samrichca I thin we do. I will address in the data hook up :) good chance to start moving some common types higher up

File: components/pages/Applications/Dashboard/Applications/index.tsx (hunk 1 above)
Comment (ciaranschutte) [reply components/pages/Applications/Dashboard/Applications/index.tsx:13]: @samrichca just temporary to build project until data hookup ticket :)

Comment (samrichca) [reply components/pages/Applications/Dashboard/Applications/index.tsx:13]: ok, understood!

**Pull request merged**